scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/remotes/postgres/PostGoalSatisfactionAction.java

```diff
@@ -0,0 +1,42 @@
+package gov.nasa.jpl.aerie.scheduler.server.remotes.postgres;
+
+import gov.nasa.jpl.aerie.scheduler.server.models.GoalId;
+import org.intellij.lang.annotations.Language;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Map;
+
+/*package-local*/ final class PostGoalSatisfactionAction implements AutoCloseable {
```

Comment:
Did we ever reach a consensus on "Post" vs "Create" nomenclature? I think we had discussed renaming everything to Create in merlin-server, but had never gotten around to it.

Reply:
We have a mix, and the reason seems to be that I naturally went for `create` when creating a new entity, such as creating a scheduling request, and `post` when inserting data that is associated with an existing thing, such as when posting scheduling results, which are associated with an existing request. Technically these "post" actions are still creating rows of data, and technically the "create" actions are posting data as well. I'm not sure if the distinction is actually beneficial, especially since it's undocumented (just in my head, I suppose). I'm happy to make things more consistent if we agree on an approach. I do feel that `insert` is the neutral ground between `create` and `post` here.

Reply:
FWIW, "post" feels more like an HTTP / REST verb than a database / Postgres verb, though maybe that's just me. "insert" feels more generally DB-y. The verb "create" feels closer to the domain for some cases, but since this Action class is an implementation detail of a Repository, maybe "create" makes more sense for the method on the Repository, and "insert" makes more sense for the low-level actions.

My two cents 😅

Reply:
It seems that we generally lean toward "insert" here, so I've gone ahead and changed the "Post" actions over to "Insert" actions.